ext/sqlite3/sqlite3.stub.php

```diff
@@ -70,6 +70,9 @@ function enableExceptions(bool $enableExceptions = false) {}
 
     /** @return bool */
     function enableExtendedResultCodes(bool $enable = true) {}
+
+    /** @return bool */
+    function setAuthorizer($callback) {}
```

Comment:
I think we should declare the parameter type:
```suggestion
    function setAuthorizer(callable $callback) {}
```